Dataset.py

```diff
@@ -694,6 +743,13 @@ def get_all_tags(self):
 
   def get_total_num_seqs(self):
     """
+    TODO: this function is only used in DataSet and MetaDataset, so can be protected
```

Comment:
No, as explained, this is used via external usage, not via `self`, so it should be public.

Reply:
Also, it's used in other places as well.

Reply:
Also, it's used in more places.